### PR TITLE
Show the rainbow donate button in some cases (1/6 ratio)

### PR DESCRIPTION
**Pull request by daschuer**
for a little bit of surprise ...

### CODE REVIEW

File: src/dialog/dlgabout.cpp [hunk 1]
```diff
@@ -343,11 +343,14 @@ DlgAbout::DlgAbout(QWidget* parent) : QDialog(parent), Ui::DlgAboutDlg() {
                                     .name(),
                             MIXXX_WEBSITE_URL,
                             tr("Official Website")));
-
-    if (!Color::isDimColor(palette().text().color())) {
-        btnDonate->setIcon(QIcon(":/images/heart_icon_light.svg"));
+    if (std::rand() % 6) {
```
Comment (Holzhaus) [inline src/dialog/dlgabout.cpp:346]: This isn't 1/6 chance, this is 1/RAND_MAX chance, or did I miss something?

Comment (Holzhaus) [reply src/dialog/dlgabout.cpp:346]: Ah no, mod 6 should work. I was confused ;-)

**Review by Holzhaus — COMMENTED** [review]
@ronso0 any comments?

**Comment by ronso0** [comment]
double-check the SVG after optimizing, it's not 100% reliable

**Comment by daschuer** [comment]
OK, I have stripped the svg a bit more.

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Pull request merged**